pepper-apis/src/main/java/org/broadinstitute/ddp/service/KitCheckService.java

```diff
@@ -406,4 +449,8 @@ public void add(KitCheckResult other) {
             }
         }
     }
+
+    <R, X extends Exception> R withAPIsTxn(HandleCallback<R, X> callback) throws X {
+        return TransactionWrapper.withTxn(TransactionWrapper.DB.APIS, callback);
+    }
```

Comment:
I had to do this thing of wrapping the open transaction call so we can mock it in the tests.